net.go

```diff
@@ -1359,6 +1369,7 @@ func (m *Memberlist) sendPingAndWaitForAck(a Address, ping ping, deadline time.T
 	}
 
 	if msgType != ackRespMsg {
+		//nolint: staticcheck
 		return false, fmt.Errorf("Unexpected msgType (%d) from ping %s", msgType, LogConn(conn))
```

Comment:
What is the error/rule being ignored here? 
Also, let's remove the space after the colon (`:`).